internal/checkconfig/checkconfig.go

```diff
@@ -170,13 +170,24 @@ func LoadConfig(configPath string) (*Config, error) {
 				if !ok {
 					log.Println("Didn't find 'kubeconfig' in 'data-gatherers.k8s/pods' configuration. Assuming it runs in-cluster.")
 				}
+				excludedNamespaces := []string{}
+				excludedNamespacesFromFile, ok := dataGathererConfigMap["exclude-namespaces"].([]interface{})
+				if ok {
```

Comment:
Probably best to invert this to check for `if !ok` and throw an unexpected error if it is not okay, which also saves the extra indenting here for the 'happy path'?

Reply:
This isn't a required field so we don't need to error. Though I should probably error on the inner `if ok`  as that implies someone has tried to specify a namespace for exclusion but it can't be parsed.

Reply:
Yeah, `!ok` indicates that the typing of the field is wrong, i.e. the user has specified: `excludedNamespaces: "sensitive-ns"` or something. Probably safest to error in these cases as we may do something unexpected (i.e. not exclude the `sensitive-ns` namespace like the use attempted to do)

Reply:
I've updated this to work better.